stix2/observables.py

```diff
@@ -255,14 +453,23 @@ class URL(_Observable):
     }
 
 
+class UNIXAccountExt(_STIXBase):
+    _properties = {
+        'gid': IntegerProperty(),
+        'groups': ListProperty(StringProperty),
+        'home_dir': StringProperty(),
+        'shell': StringProperty(),
+    }
```

Comment:
Should have an object constraint - all 4 properties are optional but at least one of them must be present.